test/functional/tests/basic/test_basic.py

```diff
@@ -30,46 +30,45 @@
 @pytest.mark.CI
 def test_cas_version():
     """
-    title: Test for CAS version
+    title: Test for version number
     description:
-        Check if CAS print version cmd returns consistent version with version file
+        Check if version printed by cmd returns consistent value with version file
     pass criteria:
-      - casadm version command succeeds
-      - versions from cmd and file in /var/lib/opencas/cas_version are consistent
+      - Version command succeeds
+      - Versions from cmd and file in /var/lib/opencas/cas_version are consistent
     """
 
-    with TestRun.step("Read cas version using casadm cmd"):
+    with TestRun.step("Read version using casadm cmd"):
         output = casadm.print_version(output_format=OutputFormat.csv)
         cmd_version = output.stdout
         cmd_cas_versions = [version.split(",")[1] for version in cmd_version.split("\n")[1:]]
 
-    with TestRun.step(f"Read cas version from {version_file_path} location"):
+    with TestRun.step(f"Read version from {version_file_path} location"):
         file_read = read_file(version_file_path).split("\n")
         file_cas_version = next(
             (line.split("=")[1] for line in file_read if "CAS_VERSION=" in line)
         )
 
     with TestRun.step("Compare cmd and file versions"):
         if not all(file_cas_version == cmd_cas_version for cmd_cas_version in cmd_cas_versions):
-            TestRun.LOGGER.error(f"Cmd and file versions doesn`t match")
+            TestRun.LOGGER.error(f"Cmd and file versions doesn't match")
 
 
 @pytest.mark.CI
 @pytest.mark.require_disk("cache", DiskTypeSet([DiskType.nand, DiskType.optane]))
 def test_negative_start_cache():
     """
-    title: Test start cache negative on cache device
+    title: Negative test for starting cache
     description:
-        Check for negative cache start scenarios
+        Check if it is forbidden to start cache using the same device or cache ID twice
```

Comment:
Pass criteria are in another section
```suggestion
        Check starting cache using the same device or cache ID twice
```